api/models/InventoryItem.js

```diff
@@ -0,0 +1,27 @@
+const mongoose = require('mongoose')
+const Schema = mongoose.Schema
+
+const InventoryItemSchema = new Schema(
+  {
+    price: {
+      type: Number,
+      required: true
+    },
+    stock: {
+      type: Number
+    },
+    category: {
```

Comment:
do you want to define some enums that this can be a category of?

e.g.

```js
const ITEM_CATEGORIES = {
  DRINK: 'DRINK',
  CANDY: 'CANDY',
  // etc
}
```

Reply:
I put "Updtated" because I created another PR with the same name, but I pushed the wrong code. I closed that PR and created a new one...put "updated" so that people don't confused between them.

Reply:
Because that PR was never merged, it is ok to remove this Updated. Most will not see that closed one and the `Updated` will only further confuse. Me wondering why it was there is an example of this.

Reply:
@evanugarte @aashrav 
should the ```stock``` be a ```String```?  Because as a ```String``` we can label it either with number like ```1,2,3,...``` or ```out of stock``` or ```check with store...```

Or should we make a new one and call it ```status```?

Reply:
Let's make stock a number, and just handle a zero value in the frontend e.g.

```js
if (stock === 0)
  // do sum about it
```